ext/oci8/tests/bug70700.phpt

```diff
@@ -0,0 +1,179 @@
+--TEST--
+Bug #70700 (LOB read loop ends early for multibyte strings)
```

Comment:
Since bug @70700 is not a code bug perhaps the title should just say 'Tests for LOBS with multibyte strings'?